widgets/src/text_input.rs

```diff
@@ -650,6 +650,13 @@ impl TextInputRef {
         }
         None
     }
+
+    pub fn returned(&self, actions: &Actions) -> Option<String> {
+        if let TextInputAction::Return(val) = actions.find_widget_action_cast(self.widget_uid()) {
+            return Some(val);
+        }
+        None
+    }
     
     pub fn return_key(&self, actions: &Actions) -> Option<String> {
```

Comment:
what is the plan with this function (it has the same implementation as the new one)?

Reply:
Ahh good call.. two people adding the same function at almost the same time. I will discuss with @makepaddev to keep only one version